eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php

```diff
@@ -178,7 +178,9 @@ public function removeURLAliases(array $urlAliases)
      */
     public function lookup($url)
     {
-        $cacheItem = $this->cache->getItem('ez-urlAlias-url-' . str_replace('/', '_', $url));
+        $cacheItem = $this->cache->getItem(
+            'ez-urlAlias-url-' . str_replace(['/', ':', '(', ')', '@'], ['_S', '_C', '_B', '_B', '_A'], $url)
```

Comment:
Note: it's not the first time we have this issue, maybe we should consider making some common cache tags/keys sanitizer.
But for now :+1:

Reply:
yes, perhaps 

We can move this to abstract for places that will need it
For instance if we face the need to re introduce support for using these characters in remote ids.